src/screens/About/Team/Team.style.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { Paragraph } from 'components/Markdown';
+import Grid from '@material-ui/core/Grid';
+import { COLOR_MAP } from 'common/colors';
+
+export const Headshot = styled.img`
+  border-radius: 50%;
+  max-width: 4rem;
+  max-height: 4rem;
```

Comment:
Would setting this as width fix the case when the images aren't properly sized? We should probably just find a way to make sure all the images are the same size